File: cfgs/pythoncfgs/FIND_DIFFERENCE_BETWEEN_SUMS_OF_TWO_DIAGONALS_1.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;conditional;foo < foo;10;
4;statement;return fun_1(foo_1 - foo_2);13;
5;statement;foo_1 = foo_2 + foo_3[foo_4][foo_5];11;
6;statement;foo_1 = foo_2 + foo_3[foo_4][foo_5 - foo_6 - 1];12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;6;
5;3;
6;

File: cfgs/java_cfgs_foo/FIND_DIFFERENCE_BETWEEN_SUMS_OF_TWO_DIAGONALS_1.java
Nodes
1;special;;;
2;statement;int d1 = 0;14;
3;statement;int d2 = 0;14;
4;statement;int i = 0;15;
5;conditional;foo_1 < foo_2;16;
6;statement;d1 += foo_1[foo_2][foo_3];18;
7;statement;d2 += foo_1[foo_2][(foo_3 - foo_4) - 1];19;
8;statement;i++;17;
9;statement;return fun_1(foo_1 - foo_2);21;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;9;
5;7;
6;8;
7;5;
8;10;
9;
